spec/menu_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'menu'
+
+describe Menu do
+
+  it 'allows to display dishes as a well formatted list' do
+    subject.add_dish("Prosciutto",12)
+    expect(subject.display_menu).to eq('pizza: £12, burger: £10, Prosciutto: £12')
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)